docs/_instructions/testterminate.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+class: Test
+title: -testterminate BOOLEAN
+summary: Do not terminate the remote framework after running the test suites.
```

Comment:
As above, I think the description should be "Terminate the remote...".

Reply:
fixed